Documentation/ceph-advanced-configuration.md

```diff
@@ -467,7 +467,7 @@ Two changes are necessary to the configuration to enable this capability:
 
 ### Use hostNetwork in the rook ceph cluster configuration
 
-Enable the `hostNetwork` setting in the [Ceph Cluster CRD configuration](https://rook.io/docs/rook/master/ceph-cluster-crd.html#samples).
+Enable the `hostNetwork` setting in the [Ceph Cluster CRD configuration](ceph-cluster-crd.md#samples).
```

Comment:
Looks good. Once these merge, it might be worth double checking that the links updated from full-url to .md reference didn't break.

Reply:
This is the same pattern that all the docs under `Documentation/` follow, but sounds good to double check.